### PR TITLE
reviewer suggestions

### LINKED ISSUE #91: Handle ANS Reviewer comments
This issue can be closed with a PR that updates the reservoir computing paper in a manner that handles the review comments below. 

### Reviewer 1
- [ ] In Fig. 3, it appears that there are two local minima (rho = 0.9, noise = 10^-4; rho = 1.3, noise = 10^-2) which are both close to the global minimum. How did you select between these two? Since the two input parameters vary quite a bit between these two, I suspect that the output of the model using the other minimum would be significantly different, too.
- [ ] What more quantitative metrics could be used to evaluate the quality of forecasts? How many Lyapunov times are useful before some accuracy threshold is reached?

### Reviewer 2
- [ ] Interesting work. Might help to clearly state the objectives of this initial phase of research towards the beginning of the paper.

### Reviewer 3
- [ ] The summary is well written but needs some discussion on verification of the proposed methodology.
- [ ] What is the basis for selection of hyperparameters and some sensitivity and parametric analysis needs to be done in order to ensure confidence in results.
- [ ] How dependent is the methodology and results on the training length and network size ?

### Submit
- [ ] A suitably revised summary must be uploaded no later than August 17 into the same system that you submitted your original paper (https://epsr.ans.org/meeting/?m=308). If your revised summary is not received by the deadline date, your original summary will be published as is.

### PR DESCRIPTION
**Pull request by samgdotson**
This PR adds some of the reviewer suggestions.

Closes #91

### CODE REVIEW

File: publications/abstracts/reservoir-computing-abs/background.tex [hunk 1]
```diff
@@ -46,4 +46,10 @@ \section{Background}
 their simplicity is balanced by the need for carefully chosen hyperparameters
 for the desired task \cite{lukosevicius_practical_2012}.
 Accurate renewable energy forecasts with \glspl{ESN} could enable new load
-following strategies for nuclear power plants.
+following strategies for nuclear power plants. The required prediction accuracy
+is related to the kinetics of a specific reactor. Each reactor design has a
```
Comment (katyhuff) [inline publications/abstracts/reservoir-computing-abs/background.tex]:
```suggestion
is related to the kinetics and dynamics of the specific reactor. Each reactor design has
```

File: publications/abstracts/reservoir-computing-abs/background.tex [hunk 2]
```diff
@@ -46,4 +46,10 @@ \section{Background}
...
+characteristic response time that depnds on neutron generation and xenon
```
Comment (katyhuff) [inline publications/abstracts/reservoir-computing-abs/background.tex]:
```suggestion
characteristic response times that depend on, for example, their mean neutron generation 
time ($\Lambda$) or xenon
```

File: publications/abstracts/reservoir-computing-abs/background.tex [hunk 3]
```diff
@@ -46,4 +46,10 @@ \section{Background}
...
+transient times. We are interested in predictions that are far enough in the
```
Comment (katyhuff) [inline publications/abstracts/reservoir-computing-abs/background.tex]:
```suggestion
transient behavior. We are interested in predictions that are far enough in the
```

File: publications/abstracts/reservoir-computing-abs/methodology.tex [hunk 4]
```diff
@@ -87,7 +87,11 @@ \subsection{\textbf{Constructing the \gls{ESN}}}
 Figure \ref{fig:gridsearch}. This search determined the optimal combination of
 spectral radius ($\rho$) and noise injection (for regularization of reservoir
 neurons), shown in Figure \ref{fig:gridsearch}, following the recommendations
-from \cite{lukosevicius_practical_2012}.
+from \cite{lukosevicius_practical_2012}. By inspecting Figure
```
Comment (katyhuff) [inline publications/abstracts/reservoir-computing-abs/methodology.tex]:
```suggestion
from \cite{lukosevicius_practical_2012}. Figure
```

File: publications/abstracts/reservoir-computing-abs/methodology.tex [hunk 5]
```diff
@@ -87,7 +87,11 @@ \subsection{\textbf{Constructing the \gls{ESN}}}
...
+\ref{fig:gridsearch} it appears that there may be multiple combinations of
```
Comment (katyhuff) [inline publications/abstracts/reservoir-computing-abs/methodology.tex]:
```suggestion
\ref{fig:gridsearch} shows that there may be multiple combinations of
```

**Review by katyhuff — CHANGES_REQUESTED** [review]

**Review by katyhuff — APPROVED** [review]
